docs/be-dr-dre.md

```diff
@@ -0,0 +1,93 @@
+
+??? tip "Regular week"
+    Usually most of this work boils down to running
+    ```bash
+    dre vote
+    ```
+
+### 2. **Review alerts for our clusters**
+
+* All alerts that our clusters send are aggregated in our [jira ops board](https://dfinity.atlassian.net/jira/ops/teams/og-a6d6c0d5-2641-4c54-8a2c-5860ef5e8f53/alerts?view=list&query=responders%3A+og-a6d6c0d5-2641-4c54-8a2c-5860ef5e8f53)
+* Heartbeats are present [here](https://dfinity.atlassian.net/jira/ops/teams/og-a6d6c0d5-2641-4c54-8a2c-5860ef5e8f53/heartbeats)
+
+??? tip "What should I do if there are alerts?"
+    
+    - It's not expected that every alert can be resolved immediately or by a single team member.
+    - The key objective is to maintain cluster stability.
+    - Evaluate the alert based on its severity and the affected cluster to determine if further action is required.
+    - Escalate or address issues as needed to ensure operations continue smoothly.
+
+### 3. **Answer in our team's slack channel**
+    
+* [`#eng-dre`](https://dfinity.enterprise.slack.com/archives/C05LD0CEAHY): General channel for activities
+* [`#eng-release`](https://dfinity.enterprise.slack.com/archives/C01DB8MQ5M1): Questions related to release process
+* [`#eng-observability`](https://dfinity.enterprise.slack.com/archives/CGZ4YGN4S): Questions related to our observability
+
+??? question "But I don't know the answers to all questions"
+
+    - It’s perfectly fine not to have all the answers.
+    - Take the initiative to investigate the issue and see how you can assist.
+    - If you’re unable to resolve the question, redirect it to the appropriate team member.
+    - The primary goal is to support the organization and relieve pressure on the rest of the team during your on-call week.
+
+### 4. **Submit requested proposals**
+
+* Replace dead nodes
+* Help in on-boarding or off-boarding of datacenters and node providers
+* Any other requested proposals
+
+??? tip "Tooling"
+    For all regular ops we have sufficient tooling implemented in our `dre` tool. For all new proposals and specific scenarios it is your responsibility to add them to the tooling as the new use cases come
+
+### 5. **Monitor the Status of CI**  
+
+- **Weekly Dependency Upgrade Job**:  
+
+   - A [GitHub Action](https://github.com/dfinity/dre/actions/workflows/update-dependencies.yaml) runs weekly to automatically upgrade dependencies.  
+   - While some weeks result in straightforward updates, others may require manual intervention due to API changes or other breaking updates.  
+
+- **Your Responsibility**:  
+
+   - Review and address any issues with the generated pull request.  
+   - Ensure the fixes are implemented and attempt to merge the PR into the repository.  
+   - Maintaining compatibility between the IC repo and our repo reduces friction and ensures our tooling operates smoothly.  
+
```

Comment:
I would like to add also:
- Mainnet topology proposals, such as `dre network --heal --optimize --ensure-operator-nodes-unassigned --ensure-operator-nodes-assigned --remove-cordoned-nodes` or a subset of these operations. The operations are still not polished enough to be run automatically.
- Provider reward adjustment proposals, if any are needed that week. Please ask in #eng-dre if you don't know if any are needed.